h2o-assemblies/main/build.gradle

```diff
@@ -16,7 +16,7 @@ configurations {
 // Dependencies
 dependencies {
     compile project(":h2o-app")
-    runtime project(":h2o-jetty-8")
```

Comment:
@Pscheidl I would keep the default as Jetty 8 in this PR and only enable it in CDH 6.